doma-spring-boot-samples/doma-spring-boot-sample-testcontainers/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# Development configuration using TestContainers
+spring.sql.init.mode=always
+
+# Doma configuration
+doma.naming=SNAKE_LOWER_CASE
+logging.level.org.springframework.jdbc.datasource.DataSourceTransactionManager=DEBUG
```

Comment:
`DataSourceTransactionManager` is old. Use `JdbcTransactionManager` instead.

`logging.level.org.springframework.jdbc.support.JdbcTransactionManager=DEBUG`